.bash_prompt

```diff
@@ -164,7 +164,14 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  # remote_branch="origin/$branch"
+  # Compute the remote tracking branch, if no remote branch then echo 0
+  remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2>/dev/null )"
```

Comment:
With respect to testing: Sure thing, we can help you out with tests. I will get to them within a week. For reference, we are using `bash` as our testing framework and Travis CI as our integration. Their setup can be seen here:

- https://github.com/twolfson/sexy-bash-prompt/blob/0.26.7/.travis.yml#L11
- https://github.com/twolfson/sexy-bash-prompt/blob/0.26.7/Makefile#L31
- https://github.com/twolfson/sexy-bash-prompt/blob/0.26.7/test/run.sh

With respect to `origin` as remote, `git` works fine with no upstream tracking. It assumes `origin` by default and my personal experience confirms that behavior (it might require setting `push.default` but that's a plausible case). Here's the relevant `man git-push` section:

> When the command line does not specify where to push with the `<repository>` argument, `branch.*.remote` configuration for the current branch is consulted to determine where to push. If the configuration is missing, it defaults to `origin`.